tests/src/system/basic/CLIActionTests.java

```diff
@@ -175,9 +175,16 @@ public void incorrectActionInvoke() throws Exception {
         String payload = "bob";
         String[] cmd = { "action", "invoke", "/whisk.system/samples/helloWorld", payload };
         RunResult rr = wsk.runCmd(cmd);
-        assertTrue("Expect a cli error exit code", rr.exitCode == 2);
-        assertTrue("Expect a cli usage message", rr.stderr.contains("usage: wsk [-h] [-v]"));
-        assertTrue("Expect a cli error message", rr.stderr.contains("wsk: error: unrecognized arguments: " + payload));
+
+        if (usePythonCLI) {
+            assertTrue("Expect a cli error exit code", rr.exitCode == 2);
+            assertTrue("Expect a cli usage message", rr.stderr.contains("usage: wsk [-h] [-v]"));
+            assertTrue("Expect a cli error message", rr.stderr.contains("wsk: error: unrecognized arguments: " + payload));
+        } else {
+            assertTrue("Expect a cli error exit code", rr.exitCode == 1);
+            assertTrue("Expect a cli usage message", rr.stderr.contains("Run 'wsk --help' for usage."));
+            assertTrue("Expect a cli error message", rr.stderr.contains("error: Invalid argument list."));
```

Comment:
Similar comment made in the Go code.  Error message when too many argument provided could be more specific.